bikeshed/__init__.py

```diff
@@ -1248,6 +1262,7 @@ def preprocess(self):
         processHeadings(self, "all") # again
         addTOCSection(self)
         addSelfLinks(self)
+        processBiblioLinks(self) # again, as biblio links could be added by addIndexSection().
```

Comment:
No, not this change. We'd need to rework `processAutoLinks` a bit.

If we move `processBiblioLinks` down after `processAutoLinks`, then the latter grabs all the biblio links and attempts to resolve them against the anchors list. Since there's no anchor named HTML5, it throws up it's hands in disgust and emits a fatal error.

We could exclude biblio links from the `findAll` call at the top of `processAutoLinks`, I suppose. I'll try that to see how it looks.